streams/shared/src/main/scala/zio/stream/experimental/ZStream.scala

```diff
@@ -82,6 +82,46 @@ abstract class ZStream[-R, +E, +O](
   ): ZStream[R1, E1, O1] =
     ZStream.absolve(ev(self))
 
+  /**
+   * Applies an aggregator to the stream, which converts one or more elements
+   * of type `A` into elements of type `B`.
+   */
+  def aggregate[R1 <: R, E1 >: E, O1 >: O, P](sink: ZTransducer[R1, E1, O1, P]): ZStream[R1, E1, P] =
+    aggregateManaged(ZManaged.succeedNow(sink))
+
+  /**
+   * Applies a managed transducer to the stream, converting elements of type `A` into elements of type `B`.
+   */
+  final def aggregateManaged[R1 <: R, E1 >: E, O1 >: O, P](
```

Comment:
I think we can just do the `aggregate` variant - a transducer already has a `ZManaged` built into it.